ipatests/test_integration/test_replica_promotion.py

```diff
@@ -570,3 +570,36 @@ def test_replica_install_with_existing_entry(self):
         master.run_command(arg)
 
         tasks.install_replica(master, replica)
+
+
+ass TestRecomendedReplicationAgreement(IntegrationTest):
+    """Maximum recomended number of agreements per replica is 4.
+    This test checks if number of agreement exceeds the recomendation,
+    warnning should be given to the user
+
+    related ticket : https://pagure.io/freeipa/issue/6533"""
+
+    num_replicas = 5
+
+    def test_recomended_replication_agreement(self):
+        master = self.master
+        replica1 = self.replicas[0]
+        replica2 = self.replicas[1]
+        replica3 = self.replicas[2]
+        replica4 = self.replicas[3]
+        replica5 = self.replicas[4]
+
+        tasks.install_master(master)
+        tasks.install_replica(master, replica1)
```

Comment:
I guess you may also use something like: 

`tasks.install_topo('star', self.master, self.replicas, [],
                           setup_replica_cas=False)`

which would be much simpler...

Reply:
thanks, it is much simple.